### ISSUE TITLE
[FEATURE REQUEST] Print all files that are read by GEOS-Chem to the log file.

### ISSUE THREAD

**Issue by yantosca** [geoschem/geos-chem#106]
## Overview:
This is related to https://github.com/geoschem/geos-chem-cloud/issues/25.  

We are trying to write a Python script to parse GEOS-Chem log files to determine which files need to be downloaded e.g. from S3 to an EBS volume.  But some GEOS-Chem routines (e.g. HEMCO extensions, UCX) do not print out the name of each file that is being opened.

## Proposed solution:
Print each file name that is being read such as "Reading /path/to/my/file.nc" or "Opening /path/to/my/file.nc".  This is what the Python script will look for.

**Comment by yantosca** [comment]
Added the following updates:

1. Print full paths for all files read by FAST-JX.  These are mostly lookup tables etc.
```
FAST-JX (RD_XXX): Reading /n/holylfs/EXTERNAL_REPOS/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/FAST_JX/v2019-06/FJX_spec.dat
FAST-JX (RD_MIE): Reading /n/holylfs/EXTERNAL_REPOS/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/FAST_JX/v2019-06/jv_spec_mie.dat
FAST-JX (RD_AOD): Reading /n/holylfs/EXTERNAL_REPOS/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/FAST_JX/v2019-06/so4.dat
FAST-JX (RD_AOD): Reading /n/holylfs/EXTERNAL_REPOS/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/FAST_JX/v2019-06/soot.dat
FAST-JX (RD_AOD): Reading /n/holylfs/EXTERNAL_REPOS/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/FAST_JX/v2019-06/org.dat
FAST-JX (RD_AOD): Reading /n/holylfs/EXTERNAL_REPOS/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/FAST_JX/v2019-06/ssa.dat
FAST-JX (RD_AOD): Reading /n/holylfs/EXTERNAL_REPOS/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/FAST_JX/v2019-06/ssc.dat
FAST-JX (RD_AOD): Reading /n/holylfs/EXTERNAL_REPOS/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/FAST_JX/v2019-06/h2so4.dat
FAST-JX (RD_AOD): Reading /n/holylfs/EXTERNAL_REPOS/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/FAST_JX/v2019-06/h2so4.dat
FAST-JX (RD_AOD): Reading /n/holylfs/EXTERNAL_REPOS/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/FAST_JX/v2019-06/dust.dat
FAST-JX (RD_JS_JX): Reading /n/holylfs/EXTERNAL_REPOS/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/FAST_JX/v2019-06/FJX_j2j.dat
```

2. Print paths for all files read by UCX:
```
UCX (INIT_UCX): Reading /n/holylfs/EXTERNAL_REPOS/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/UCX_201904/Montreal_fix.nc
```

3. Print paths for all files read by drydep:
```
%% READ_DRYDEP_INPUTS: Reading /n/holylfs/EXTERNAL_REPOS/GEOS-CHEM/gcgrid/data/ExtData/CHEM_INPUTS/Olson_Land_Map_201203/Olson_2001_Drydep_Inputs.nc
```

This is being added into GEOS_Chem 12.6.1/

**Comment by yantosca** [comment]
This is now achieved by these commits:

1. https://github.com/geoschem/geos-chem/commit/24919372b5ed8dfc4e89b625783b1de13bc356ee
2. https://github.com/geoschem/geos-chem/commit/ad76112a0df0e02225e0f6b5e15e878fd829187d
3. https://github.com/geoschem/geos-chem/commit/62743d4dd9d325ec6a80841f29b889bdc1caba95

**Comment by yantosca** [comment]
Now included in GC 12.6.1, released on 28 Oct 2019.
